kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/offer/OfferRepository.java

```diff
@@ -60,6 +66,21 @@ public void onFailure(Throwable t) {
         });
     }
 
+    private void updateCacheOfferList(OfferList response) {
+        removeAllNoneNativeOffers();
```

Comment:
I would keep native spend offer and market place offers in 2 different list and updated each one independently. In getCachedOfferList we can simply return a contact list of the 2

Reply:
👍